staging/src/k8s.io/code-generator/cmd/client-gen/generators/generator_for_group.go

```diff
@@ -210,17 +210,12 @@ func New(c $.restRESTClientInterface|raw$) *$.GroupGoName$$.Version$Client {
 
 var setInternalVersionClientDefaultsTemplate = `
 func setConfigDefaults(config *$.restConfig|raw$) error {
-	g, err := scheme.Registry.Group("$.groupName$")
-	if err != nil {
-		return err
-	}
-
 	config.APIPath = $.apiPath$
 	if config.UserAgent == "" {
 		config.UserAgent = $.restDefaultKubernetesUserAgent|raw$()
 	}
-	if config.GroupVersion == nil || config.GroupVersion.Group != g.GroupVersions[0].Group {
-		gv := g.GroupVersions[0]
+	if config.GroupVersion == nil || config.GroupVersion.Group != scheme.Scheme.PrioritizedVersionsForGroup("$.groupName$")[0].Group {
```

Comment:
worth checking if len is 0 and erroring rather than panicing?

Reply:
> worth checking if len is 0 and erroring rather than panicing?

If we got the scheme wrong somehow?  I don't think so.  That's a meta developer error.

Reply:
ah, missed that the referenced scheme was generated of a piece. that's fine, then